app/views/result_texts/_edit.html.erb

```diff
@@ -3,7 +3,7 @@
     <%= f.text_field :name, style: "margin-top: 10px;" %><br />
     <%= f.fields_for :result_text do |ff| %>
       <div class="form-group">
-        <%= quill_editor nil, { id: 'result_result_text_attributes_text', name: 'result[result_text_attributes][text]', value: @result.result_text.text } %>
+        <%= ff.text_area(:text, class: 'tinymce', cols: 120, rows: 40, value: @result.result_text.text ) %>
```

Comment:
Maybe I'm nitpicking, but we could externalize this into our own function into `bootstrap_form_helper` (e.g. `tiny_mce_editor`), which would simply render this code.